cypress/e2e/checkBox.cy.js

```diff
@@ -19,7 +19,7 @@ describe("Check functionallity of check boxes and expand buttons", () => {
 
   it("Check the functionality of check boxes", () => {
     pageCheckBox.clickOnCheckBoxButton();
-    pageCheckBox.checkIfThereIsOnlyOneLabelWithTextHome();
+    pageCheckBox.checkIfThereIsOnlyOneLabel();
     pageCheckBox.checkInMoreLabelsAndAssert(moreLabels);
```

Comment:
what is the test name meaning?
Check in more labels and assert? It doesn't make any sense.